winterfell/config/mongoose.js

```diff
@@ -13,6 +18,7 @@ module.exports = function(environment) {
   });
   db.on('open', function () {
       console.log('Database connection opened at ' + address);
+      setupDb();
```

Comment:
So our database is going to have a new record inserted each time we startup the app? Eventually we will have a lot of duplicate records in the DB right?